Gemfile.lock

```diff
@@ -247,7 +247,7 @@ PLATFORMS
   ruby
 
 DEPENDENCIES
-  cocoapods (= 1.7.5)!
+  cocoapods (= 1.6.1)!
```

Comment:
Do we want to downgrade Cocoapods? 🤔

Reply:
Mmm... I wanted to align with WPiOS and I thought that release/13.2.x was the branch we were using there. 1.6.1 is listed in that branch, so I changed this.
Reverting to 1.7.5 now.